doc/source/tune/doc_code/faq.py

```diff
@@ -93,6 +93,30 @@ def train(config):
     tuner.fit()
     # __resources_pgf_end__
 
+    # __resources_lambda_start__
+    tuner = tune.Tuner(
+        tune.with_resources(
+            train_fn,
+            resources=lambda spec: {"GPU": 1} if config.use_gpu else {"GPU": 0},
```

Comment:
this should be spec.config.use_gpu? config is not defined here?

Reply:
@richardliaw good catch, let me just update spec to config to avoid confusion